autoload/lsp/lspserver.vim

```diff
@@ -114,7 +114,7 @@ def ServerInitReply(lspserver: dict<any>, initResult: dict<any>): void
   capabilities.ProcessServerCaps(lspserver, caps)
 
   if caps->has_key('completionProvider')
-    if opt.lspOptions.autoComplete
+    if opt.lspOptions.autoComplete || opt.lspOptions.omniComplete
       lspserver.completionTriggerChars =
 			caps.completionProvider->get('triggerCharacters', [])
```

Comment:
perhaps just remove this 'if' which to do such assignment always if it had such cap,
besides 'omni complete' seems not only depended on 'omniComplete' this option only.